opentelemetry-sdk/src/opentelemetry/sdk/util/instrumentation.py

```diff
@@ -23,6 +25,62 @@ class InstrumentationInfo:
 
     __slots__ = ("_name", "_version", "_schema_url")
 
+    @deprecated(version="1.10.0", reason="You should use InstrumentationScope")
```

Comment:
should this be version 1.11.0?

Reply:
You are correct. This should be the version it is getting deprecated in.